src/sql/migrations/1-init.sql

```diff
@@ -94,28 +96,39 @@ CREATE TABLE IF NOT EXISTS df.comment_followers
     following_comment_id bigint NOT NULL
 );
 
-CREATE INDEX IF NOT EXISTS idx_follower_account 
+CREATE TABLE IF NOT EXISTS df.schema_version
+(
+    value integer DEFAULT 1 NOT NULL
+);
+INSERT INTO df.schema_version
+(
+    -- Note that the value '1' equals to the DEFAULT of "value" field
+    SELECT 1
+    WHERE NOT EXISTS(SELECT * FROM df.schema_version)
+);
```

Comment:
Consider this:
1. Insert `0` as a default version of migration.
2. Execution of the first migration script should insert the version of successfully executed migration.